cypress/integration/appbar/collapsible-app-bar.spec.js

```diff
@@ -0,0 +1,14 @@
+/// <reference types="cypress" />
+
+
+describe('Collapsible app bar', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:3000/collapsible');
+    });
+
+    it('show tool bar and banner', () => {
+        cy.get('#scroll-area > .MuiList-root').scrollIntoView({ duration: 1000, offset:{ top: 0, left: 0 }})
+        cy.get('.MuiPaper-root > .MuiToolbar-root').should('be.visible')
+        cy.get('.makeStyles-banner-340').scrollIntoView().should('be.visible')

```

Comment:
this is a JSS autogenerated class name...is it always going to be .makseStyles-banner-340? I'm thinking if we were to add another class in the code this might change. Might be more resilient to use a test id